docs/content/learn/tutorials/extensions/_index.md

```diff
@@ -35,101 +35,351 @@ Finally, the optional `class` attribute should be used in cases where a property
 
 ### Using an Existing Prop
 
-This sections covers the basic use of a `prop` leveraging all of its attributes.  The OSCAL SSP metadata allows for zero or more `location` data items, each containing zero or more `props`.  In this example, an organization needs to document their primary and alternate data center locations.  This is achieved by specifying the `name`, `value`, and `class` of the properties (on lines #10 and #14) for each location.
+This sections covers the basic use of a `prop` leveraging all of its attributes.  The OSCAL SSP [metadata](/concepts/layer/overview/#metadata-overview) allows for zero or more `location` data items, each containing zero or more `props`.  The [Metadata Tutorial](/learn/tutorials/metadata/) provides in-depth description and walk-through examples of creating OSCAL metadata.
+
+In this example, an organization needs to document their primary and alternate data center locations.  This is achieved by specifying the `name`, `value`, and `class` of the properties for each location.
+
+{{< tabs XML JSON YAML >}}
 
-{{< tabs XML >}}
 {{% tab %}}
 {{< highlight xml "linenos=table" >}}
 <?xml version="1.0" encoding="UTF-8"?>
-<system-security-plan ...>
+<system-security-plan uuid="ce16b9af-6853-4abe-9e27-b79d034c0adc">
   <metadata>
     <title>OSCAL Basic Prop Example</title>
     <last-modified>2001-12-17T09:30:47Z</last-modified>
     <version>20210812</version>
     <oscal-version>1.0.0</oscal-version>
     <location uuid="location uuid">
       <title>....</title>
-      <prop name="type" value="data-center" class="primary">
+      <prop name="type" value="data-center" class="primary">...</prop>
     </location>
     <location uuid="location uuid">
       <title>....</title>
-      <prop name="type" value="data-center" class="alternate">
+      <prop name="type" value="data-center" class="alternate">...</prop>
     </location>
   </metadata>
-  <import-profile .../>
-  <system-characteristics .../>
-  <system-implementation .../>
+  <import-profile>...</import-profile>
+  <system-characteristics>...</system-characteristics>
+  <system-implementation>...</system-implementation>
 </system-security-plan>
 {{< /highlight >}}
+
+The `@name` attribute is set to "type" which is an OSCAL pre-defined property. The `@value` attribute is set to "data-center" which is an OSCAL pre-defined value for location "type" properties. The `@class` attribute in this case is used to indicate a subclass of data-center, and is set the the appropriate OSCAL pre-defined values for data center location properties.  Note that in this example, the `@ns` attribute is omitted because we are using a standard OSCAL defined `<prop>` element.
```

Comment:
```suggestion
The `@name` attribute is set to "type" which is an OSCAL pre-defined property for a `<location>`. The `@value` attribute is set to "data-center" which is an OSCAL pre-defined value for location "type" properties. The `@class` attribute in this case is used to indicate a subclass of data-center, and is set the the appropriate OSCAL pre-defined values for data center location properties.  Note that in this example, the `@ns` attribute is omitted because we are using a standard OSCAL defined `<prop>` element.
```

Maybe link the "pre-defined" to the related docs.